File: Features/queryablestate-carinsurance/contracts/src/test/java/net/corda/samples/carinsurance/states/ClaimTests.java
package net.corda.samples.carinsurance.states;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class ClaimTests {

    private final String desc = "claim description: my car was hit by a blockchain";
    private final String claimNumber = "B-132022";
    private final int claimAmount = 3000;

    @Test
    public void constructorTest() {
        Claim st = new Claim(claimNumber, desc, claimAmount);

        assertEquals(claimNumber, st.getClaimNumber());
        assertEquals(desc, st.getClaimDescription());
        assertEquals(claimAmount, st.getClaimAmount());
    }

}

File: kotlin-samples/Features/customquery-carinsurance/contracts/src/test/kotlin/net/corda/samples/carinsurance/states/ClaimTests.kt
package net.corda.samples.carinsurance.states

import org.junit.Assert
import org.junit.Test

class ClaimTests {
    private val desc = "claim description: my car was hit by a blockchain"
    private val claimNumber = "B-132022"
    private val claimAmount = 3000

    @Test
    fun constructorTest() {
        val (claimNumber1, claimDescription, claimAmount1) = Claim(claimNumber, desc, claimAmount)
        Assert.assertEquals(claimNumber, claimNumber1)
        Assert.assertEquals(desc, claimDescription)
        Assert.assertEquals(claimAmount.toLong(), claimAmount1.toLong())
    }
}
